indico_themes_canonical/static/css/_default/header.css

```diff
@@ -5,31 +5,70 @@ Wraps around the upper part of the conference header.
 */
 .confheader {
   background: rgb(0, 0, 0);
-  background: linear-gradient(90deg, rgba(0, 0, 0, 1) 17%, rgba(233, 84, 32, 1) 98%);
   color: #fff;
   overflow: hidden;
   position: relative;
   height: 399px;
 }
 
+.confheader div {
+  color: var(--brand-white) !important;
+}
+
 /*
 Wraps around the title box
 */
 .confTitleBox {
   background: none;
   display: flex;
   margin: auto;
-  padding: 80px 20px;
+  padding: 80px 20px 0 20px;
   position: relative;
   text-align: left;
   width: var(--full-width);
 }
 
+/*
+Wraps around the subtitle box
+*/
+.confSubTitleBox {
+  background: none;
+  border: 0;
+  padding: 0px 20px;
+  width: var(--full-width);
+  margin: auto;
+}
+
+.confSubTitleBox .search-box {
+  display: none;
+}
+.confSubTitleContent {
+  border-left: 4px solid var(--brand-primary);
+  display: inline-block;
+  margin-left: var(--global-margin-left);
+  width: 100%;
+  padding-left: 1rem;
+}
+
+/*
+Event Logo placement
+*/
+
+.confLogoBox {
+  position: relative;
+}
+
+.confTitleBox .confLogo {
+  position: absolute;
```

Comment:
If you have a smarter way to place the built-in logo I'm all for it. I wanted to avoid mucking around with javascript and this seemed the next best option.

Reply:
Given how static the header is, I think this is a good solution.